### ISSUE TITLE
✨ Admin / Communities / mostly Styling update

### ISSUE THREAD

**Issue by Jerryescandon** [GCTC-NTGC/gc-digital-talent#11424]
## ✨ Feature

Update styling of the Community pages, standardize the way things work, and bring it in line with the overall platform's look and feel. And add a new "mandate authority" field to communities.

### Out of scope
Don't build this even if it's in the design

- [ ] Updates to the table component
- [ ] Platform admin navigation "horizontal nav"
- [ ] Table column "Status"
- [ ] Advanced tools tab
- [ ] Updates to the mobile tab component

## 🎨 Design

[Figma file](https://www.figma.com/design/3VPSAv5689GK51Us6BxbJm/Admin---Communities?node-id=10068-3706&t=uPsqIXspXLYVDGsi-1) by @Jerryescandon 

## ✅ Acceptance Criteria

- [ ] New field (English & French) for "mandate authority" in community model: **String, nullable (not required)**

- [ ] Communities page (/admin/communities)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
  - [ ] Add column for "mandate authority"
- [ ] Create community (/admin/communities/create)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
  - [ ] Add "mandate authority"
- [ ] Community information (/admin/communities/:id:)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
  - [ ] Add "mandate authority"
- [ ] Edit Community (/admin/communities/:id:/edit)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
  - [ ] Uses same UI as "create page"
  - [ ] Add "mandate authority" 
- [ ] Manage access (tab) (/admin/communities/:id:/manage-access)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
  - [ ] Add, edit & remove members links and dialogues styling (if needed) **NOTE:** the dropdown menu for Actions is a table component update and is out of scope for this issue
  - [ ] Column order on the table: Name (first), Actions (second), Contact email address, Membership roles

- [ ] Mobile view for everything

- [ ] Design Review by @Jerryescandon 

## Blockers
```[tasklist]
### Blocked by
- [ ] #11344 
```

**Comment by gobyrne** [comment]
@Jerryescandon we only wanna use "keys" if there's a good reason to. Does it make sense for communities?

**Comment by gobyrne** [comment]
Devs have asked to break off "Archive" work into another ticket.

**Comment by Jerryescandon** [comment]
Add mandate authority, OCIO
